src/jmclient/support.py

```diff
@@ -352,14 +365,18 @@ def calc_zero_change_cj_amount(ordercombo):
         cjamount = int(cjamount.quantize(Decimal(1)))
         return cjamount, int(sumabsfee + sumrelfee * cjamount)
 
+    fee_per_kb = jm_single().bc_interface.estimate_fee_per_kb(
+        jm_single().config.getint("POLICY", "tx_fees"), randomize=False)
```

Comment:
Is there an argument for throwing in a 20% tolerance here, i.e. if 2 participants are *trying* to use roughly the same fee estimate (e.g. 10 blocks target), and just happen to be slightly different, then let's make it more likely that they'll succeed to negotiate?